README.md

```diff
@@ -43,6 +43,14 @@ In both cases include migration path in issue's description (and later in releas
 6. Add the version with change log to [GitHub Releases page](https://github.com/visionappscz/bootstrap-ui/releases)
 7. Deploy docs to production (outside of GitHub)
 
+## Don’t Forget
+
+* `demo.bootstrap-ui.com`
+    * Update package version in `package.json` 
+* `www.bootstrap-ui.com`
+    * Update package version in `package.json`
+    * Update number in button &ldquo;WHAT’S NEW IN 3.0&rdquo; 
```

Comment:
This is a new occurence of version number that needs to be maintained. I’d prefer something more general so the README doesn’t need to be updated.